### ISSUE TITLE
Book content inaccessible for screenreader on Linux

### ISSUE THREAD

**Issue by RastislavKish** [edrlab/thorium-reader#2020]
On Linux, the book content area is not accessible using arrow keys, making it impossible to read.

## Steps to reproduce

1. download the latest release of Thorium, say in the AppImage format and give it execution permission.
2. run ./Thorium-2.3.0.AppImage book.epub where book.epub is any book.
3. In the open window, try to read the book content using Up/Down arrow keys.

## Expected behavior

Orca should have free access to the book content, like it was a standard webpage in the Electron window.

## Actual behavior

Orca keeps repeating either the title of the book in form of a link, or a heading Main content. The behavior is inconsistent though, at times a line from the content is randomly spoken, or focus can move further to the rest of the Thorium UI, like there was nothing in between.

Flat review can see the content of the book being present, at least it did on the first opening of a book, another try with additional Resume heading did not reveal the content, I'm not sure whether it's not there this time or it's just a flat review failure, sometimes it happens that parts of website are missing in FR even in standard browsers.

## Environment

Ubuntu Mate 22.04 64-bit, Orca gnome-44 branch, Thorium 2.3.0 (though the problem has been apparently present for longer time).

**Comment by gautierchomel** [comment]
Thank you for this feedback as we have no mean to test this usecase. 

Using the Tab key you should find the link "skip to content".

**Comment by RastislavKish** [comment]
@gautierchomel the problem seems not to be getting to the content itself, but rather moving around it using arrow keys and other means of navigation. For example, on a page of book with links and headings, the only kind of navigation that works as expected is jumping by links using the Tab / Shift+Tab shortcuts. Trying to look around using arrow keys, the H key for moving through headings, yields repeating a particular element of the page, for example a focused heading from the book, but if focusing is unsuccessful, this can also be the Main contents heading etc. It seems to me like the app was blocking the keyboard in some way, so it's not processed as it should be.

**Comment by gautierchomel** [comment]
I use linux and navigate with keyboard. A [list of shortcuts](https://thorium.edrlab.org/docs/400_ressources/402_keyboard-shortcuts/) is available from the doc website. 

The app is designed with AT support in mind. If something is blocking here it might come from interaction with Orca wich is an untested usecase. Interactions with NVDA are working and documented. I would be very happy to add documentation about using Thorium with Orca.

**Comment by RastislavKish** [comment]
@gautierchomel after doing some more testing, it seems you're actually right with the Orca bug explanation. Linux accessibility stack is undergoing some backward-breaking changes, with some point-release distros not getting the updates, Ubuntu 22.04 is one such example. I tried with 23.10 and everything seems to work fine.
I will close this as it's in-fact resolved on the newer systems, while new Ubuntu LTS will be released soon. Thanks for the feedback!
